app/src/main/java/org/thunderdog/challegram/unsorted/Settings.java

```diff
@@ -363,6 +364,7 @@ private static String key (String key, int accountId) {
   private static final int FLAG_OTHER_NO_CHAT_QUICK_REPLY = 1 << 11;
   private static final int FLAG_OTHER_SEND_BY_ENTER = 1 << 12;
   private static final int FLAG_OTHER_HIDE_CHAT_KEYBOARD = 1 << 13;
+  private static final int FLAG_OTHER_USE_QUICK_TRANSLATION = 1 << 14;
```

Comment:
Without version bump and explicit nullifying of this flag, there's a little chance that it will be enabled for some users who have Telegram X installed for several years and `SharedPreferences` upgrade was unsuccessful, as this bit was previously used (`DISABLED_FLAG_OTHER_SHOW_FORWARD_OPTIONS`). But most likely shouldn't happen. Just noting in case such report will be received.